packages/assertions/assertions-csv/manual.csv

```diff
@@ -1,109 +1,102 @@
-"ID","Status","Assertion"
```

Comment:
Is the diff this big due to addition of comment field and its associated comma?

Reply:
Yes, that's really it